app/assets/stylesheets/components/_file-input.scss

```diff
@@ -50,7 +50,7 @@
   line-height: 1.5;
   // For content to appear as vertically centered, offset the larger line-height of the banner to
   // match the space below the drag text.
-  margin-top: ((1.5rem - size('body', '2xs')) - ((1.625rem * 1.5) - 1.625rem)) / 2;
+  margin-top: ((1.5rem - size('body', '2xs')) - ((1.625rem * 1.5) - 1.625rem)) * 0.5;
```

Comment:
was this an int vs float issue?

Reply:
Now that we're on Dart Sass, we could alternatively swap it with the new math module function, as in d492148.